samples/example_expo/app/(home)/_layout.tsx

```
@@ -10,81 +9,78 @@ const config = {
+            switch (verificationMethod) {
+                case 'providerId':
+                    console.assert(config.REACT_APP_RECLAIM_APP_ID, 'RECLAIM_APP_ID is not set');
+                    console.assert(config.REACT_APP_RECLAIM_APP_SECRET, 'RECLAIM_APP_SECRET is not set');
+                    verificationResult = await reclaimVerification.startVerification({
+                        appId: config.REACT_APP_RECLAIM_APP_ID ?? '',
+                        secret: config.REACT_APP_RECLAIM_APP_SECRET ?? '',
+                        providerId: inputText,
+                    });
+                    break;
+                case 'jsonConfig':
+                    const jsonConfig = JSON.parse(inputText);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)